PythonVisualizations/SimpleQueue.py

```diff
@@ -8,60 +8,85 @@
 
 WIDTH = 800
 HEIGHT = 400
-
 CELL_SIZE = 50
 ARRAY_X0 = 100
 ARRAY_Y0 = 100
 
-class Queue(object):
+class Queue(VisualizationApp):
 
     colors = ['red', 'green', 'blue', 'orange', 'yellow', 'cyan', 'magenta',
               'dodgerblue', 'turquoise', 'grey', 'gold', 'pink']
```

Comment:
The `drawable` class provides a `palette` attribute with a set of colors that work well with black text for the values.  Using the common class definition also makes it easier to make corrections across all visualizations (e.g. if we determine that some colors don't work for some situations or decide to use a narrower range of hues).